ui/src/components/ProcessTree.scss

```
@@ -235,8 +245,11 @@
   gap: $pad-small;
   padding: $pad-small $pad-medium;
   margin-bottom: $pad-medium;
-  background-color: rgba(255, 232, 236, 0.6);
-  border: 1px solid rgba(255, 92, 131, 0.4);
+  // Neutral grey container — the page already shows alert severity via the
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)